package-lock.json

```diff
@@ -0,0 +1,662 @@
+        "npm": ">=10.0.0"
+      }
+    },
+    "app": {
+      "name": "headlamp",
+      "version": "0.36.0",
+      "extraneous": true,
+      "dependencies": {
+        "copyfiles": "^2.4.1",
+        "cross-env": "^7.0.3",
+        "dotenv": "^16.4.5",
+        "electron-wix-msi": "^5.1.3",
+        "find-process": "^1.4.10",
+        "glob": "^11.0.3",
+        "i18next": "^23.12.1",
+        "i18next-fs-backend": "^2.3.1",
+        "mkdirp": "^3.0.1",
+        "nock": "^14.0.5",
+        "shx": "^0.4.0",
+        "tar": "^7.4.3",
+        "yargs": "^17.7.2"
+      },
+      "devDependencies": {
+        "@babel/cli": "^7.24.8",
+        "@babel/core": "^7.24.9",
+        "@babel/preset-env": "^7.24.8",
+        "@babel/preset-typescript": "^7.24.7",
+        "@headlamp-k8s/eslint-config": "^0.6.0",
+        "@types/jest": "^29.5.14",
+        "@types/node": "^22.14.0",
+        "@types/semver": "^7.7.0",
+        "electron": "^31.2.0",
+        "electron-builder": "^24.13.3",
+        "i18next-parser": "^9.0.0",
+        "jest": "^29.7.0",
+        "typescript": "5.5.4"
+      },
+      "engines": {
+        "node": ">=20.11.1",
+        "npm": ">=10.0.0"
+      }
+    },
+    "app/e2e-tests": {
+      "version": "1.0.0",
+      "extraneous": true,
+      "license": "ISC",
+      "devDependencies": {
+        "@playwright/test": "^1.48.1",
+        "@types/node": "^22.7.5"
+      }
+    },
+    "e2e-tests": {
+      "version": "1.0.0",
+      "extraneous": true,
+      "license": "ISC",
+      "dependencies": {
+        "@axe-core/playwright": "^4.10.1",
+        "yaml": "^2.3.4"
+      },
+      "devDependencies": {
+        "@playwright/test": "^1.42.1",
+        "@types/node": "^20.12.2"
+      }
+    },
+    "eslint-config": {
+      "name": "@headlamp-k8s/eslint-config",
+      "version": "0.6.0",
+      "extraneous": true,
+      "license": "Apache-2.0",
+      "dependencies": {
+        "@typescript-eslint/parser": "^8.3.0",
+        "eslint": "^8.57.0",
+        "typescript": "5.5.4"
+      },
+      "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^8.3.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-import": "^2.24.2",
+        "eslint-plugin-jsx-a11y": "^6.9.0",
+        "eslint-plugin-react": "7.35.0",
+        "eslint-plugin-react-hooks": "^4.6.2",
+        "eslint-plugin-simple-import-sort": "^12.1.1",
+        "eslint-plugin-unused-imports": "^4.1.3",
+        "yaml": "^2.5.0"
+      },
+      "peerDependencies": {
+        "@typescript-eslint/eslint-plugin": "^8.3.0",
+        "eslint": "^8.57.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-import": "^2.24.2",
+        "eslint-plugin-jsx-a11y": "^6.9.0",
+        "eslint-plugin-react": "7.35.0",
+        "eslint-plugin-react-hooks": "^4.6.2",
+        "eslint-plugin-simple-import-sort": "^12.1.1",
+        "eslint-plugin-unused-imports": "^4.1.3"
+      }
+    },
+    "frontend": {
```

Comment:
Looks like the package-lock.json along with node_modules needs to be removed and regenerated?